processing/data_collection/gazette/spiders/mg_campobelo.py

```diff
@@ -0,0 +1,8 @@
+from gazette.spiders.instar_base import BaseInstarSpider
+
+
+class MgCampoBeloSpider(BaseInstarSpider):
+    TERRITORY_ID = "3111200"
+    name = "mg_campobelo"
```

Comment:
To not be spammy, I'll comment in just this spider.

Cities that have names with multiple words were added with `_` (underscore) separating those words at the city name (file name and spider name). If you have nothing against it, follow it because having different patterns can break expectations.